### PR TITLE
Correct invalid groups_attribute_path option + added details for nested groups

### PR DESCRIPTION
**Pull request by rgnvldr**
**What is this feature?**

Corrects typo for `groups_attribute_path` and adds details about how to extract groups that are nested via Keycloak

### CODE REVIEW

File: docs/sources/setup-grafana/configure-security/configure-authentication/keycloak/index.md [hunk 1]
```diff
@@ -109,7 +109,14 @@ This will add the `groups` claim to the id_token. You can then use the `groups`
 
 ```ini
 [auth.generic_oauth]
-group_attribute_path = groups
+groups_attribute_path = groups
+```
+
+If using nested groups that contain special characters such as quotes or colons, the JMESPath parser can perform a harmless reverse function so Grafana will properly evaluate nested groups. eg. a parent group named `Global` with nested group `department` containing a list of groups:
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/keycloak/index.md:115]:
```suggestion
If you use nested groups containing special characters such as quotes or colons, the JMESPath parser can perform a harmless reverse function so Grafana can properly evaluate nested groups. For example, a parent group named `Global` with nested group `department` containing a list of groups:
```

Comment (chri2547) [reply docs/sources/setup-grafana/configure-security/configure-authentication/keycloak/index.md:115]: @IevaVasiljeva , can you please take a look at this PR? If not you, who can help? Thank you!

**Review by chri2547 — COMMENTED** [review]
Added suggestion for the wording. Thank you for the contribution!

**Review by IevaVasiljeva — COMMENTED** [review]
LGTM, but cc @Jguer in case he can think of a neater way to correctly deal with special characters.

And thanks for extending the documentation @rgnvldr!

**Review by IevaVasiljeva — APPROVED** [review]

**Pull request merged**